permissions/create_iam_reader_permissions.sql

```diff
@@ -0,0 +1,7 @@
+GRANT USAGE ON SCHEMA audit TO iam_reader;
+GRANT USAGE ON SCHEMA billing TO iam_reader;
+GRANT SELECT ON audit.audit_events TO iam_reader;
+GRANT SELECT ON billing.billing_events TO iam_reader;
+GRANT SELECT ON billing.fraud_events TO iam_reader;
+-- Production and staging only
```

Comment:
There's a line missing here...

```suggestion
GRANT TEMPORARY on DATABASE events TO iam_reader;
-- Production and staging only
```

Reply:
I choose bootstrap because at some point we will be taking "snapshot" of where the database is currently for Flyway to understand how to interact with it. After that we should probably then have a migrations folder for any future changes. These first three scripts won't really be "migrations" in that instance.

Reply:
Renamed!